src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -640,6 +652,11 @@ public override Symbol VisitTypeParameter(TypeParameterSymbol symbol)
                 return otherTypeParameters[symbol.Ordinal];
             }
 
+            private TypeWithAnnotations VisitTypeWithAnnotations(TypeWithAnnotations type)
+            {
+                return TypeWithAnnotations.Create((TypeSymbol?)Visit(type.Type));
```

Comment:
>Create [](http://example.com/codeflow?start=43&length=6)

Why create and drop all additional info. Custom modifiers, for example. We also have a ```WithType``` helper or something similar, I think.

Reply:
This method is not used. Removed.